deployment/docker-compose.yaml

```diff
@@ -171,14 +173,15 @@ services:
         "--var=EMIT_METRICS_LOGS:false",
         "--var=SYNC_RATELIMIT_ON_NO_DATA:1.0",
         "--var=CLICKHOUSE_URL:http://default:password@clickhouse:8123",
-        "--var=CLICKHOUSE_INSERT_URL:http://proxy:password@chproxy:7123",
+        "--var=CLICKHOUSE_INSERT_URL:http://apiv2:7070",
+        "--var=CLICKHOUSE_PROXY_TOKEN:chproxy-test-token-123",
       ]
 
     depends_on:
       - planetscale
       - agent
       - clickhouse
-      - chproxy
+      - apiv2
```

Comment:
.github/workflows/job_test_api_local.yaml we have to adjust this as well because it now complains that it doesnt know the apiv2 container since we dont start it

Reply:
yeah I got ahead of myself, I'll fix it in the stacked PR immediately after